docker/starburst/etc/jvm.config

```diff
@@ -1,13 +1,13 @@
 -server
--Xmx1G
--XX:-UseBiasedLocking
--XX:+UseG1GC
+-XX:InitialRAMPercentage=80
```

Comment:
Probably this should go into a commit of its own to have corresponding comments.

Reply:
extracted to the separate commit